### ISSUE TITLE
Features that bind to popups should have appropriate ARIA properties

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#317]
There's currently nothing indicating to users of assistive technology that a feature can be interacted with.

Features (rendered as `<path tabindex="0">`) that bind to popups should:

- convey the `aria-expanded="true|false"` state of the corresponding popup (as suggested by @shepazu)
- potentially have `role="button"`
